netbout-client/src/main/java/com/netbout/client/RtAttachment.java

```diff
@@ -50,6 +50,9 @@
  * @author Yegor Bugayenko (yegor@teamed.io)
  * @version $Id$
  * @since 2.0
+ * @todo #1090:30min remove code duplication in public methods in this class
+ *  except in method name(). Create one or more private methods to achieve
+ *  this.
```

Comment:
@CarlosGines please rephrase this a little :)
Give the explicit names of those methods containing duplication and remove the second sentence, it kind of says nothing if you think about it :)